### PR TITLE
use direct require lodash, remove unused dependencies, add some explicit dependencies

### PR DESCRIPTION
**Pull request by rezk2ll**

### CODE REVIEW

File: src/frontend/js/modules/attachment/list/attachment-list.controller.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
-require('./attachment-list-providers.service.js');
```
Comment (tuanlc) [inline src/frontend/js/modules/attachment/list/attachment-list.controller.js:1]: Why did you remove this? `esnAttachmentListProviders` is still there, right?

Comment (rezk2ll) [reply src/frontend/js/modules/attachment/list/attachment-list.controller.js:1]: ~~It is already required in the app module.~~

File: src/frontend/js/modules/attachment/list/attachment-list.controller.js [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+const _ = require('lodash');
```
Comment (itswillta) [inline src/frontend/js/modules/attachment/list/attachment-list.controller.js:1]: I think now that we're using Webpack, perhaps we can utilize ES modules and tree shaking. The whole lodash library is quite huge. What we really use is just a fraction of it, but we're currently importing the whole library. The same goes for other utility libraries. I know now is not the time to do optimizations, but I think we should keep this in mind until then.

**Review by itswillta — DISMISSED** [review]

File: src/frontend/js/modules/attachment/list/attachment-list.controller.js (hunk 2 above)
Comment (MichaelBailly) [reply src/frontend/js/modules/attachment/list/attachment-list.controller.js:1]: Good point Huy. However we have first to update our lodash to a more recent version, because our version does not support per-method inclusion. and in more recent versions they renamed "contains" to "includes". And restangular uses "contains", as well as parts in our own code...

**Review by tuanlc — DISMISSED** [review]

File: src/frontend/js/modules/attachment/list/attachment-list.controller.js (hunk 2 above)
Comment (chamerling) [reply src/frontend/js/modules/attachment/list/attachment-list.controller.js:1]: Yes, already tried to have newer version, Restangular failed on it.

**Review by chamerling — DISMISSED** [review]

File: src/frontend/js/modules/attachment/list/attachment-list.controller.js (hunk 1 above)
Comment (rezk2ll) [reply src/frontend/js/modules/attachment/list/attachment-list.controller.js:1]: nvm i'm wrong, thanks

**Review by chamerling — APPROVED** [review]

**Review by tuanlc — APPROVED** [review]
